src/pages/join-us.js

```diff
@@ -35,8 +35,13 @@ const CasesPage = () => (
           join us
         </Title>
         <Text marginBottom="50px">
-          We have a young team and that is part of the business. This brings motivation and focus in each project,
-          making it part of a collective effort, to exceed.
+          Over the years we have developed our own way to deal and help with each other. We like to enable dreams and
```

Comment:
not better to add this already in the i18n branch?

Reply:
Yeah that branch didn't exist in the time I was working on it. 😞